libs/providers/flagd/README.md

```diff
@@ -74,6 +73,19 @@ Flag configurations for evaluation are obtained via gRPC protocol using [sync pr
 
 In the above example, the provider expects a flag sync service implementation to be available at `localhost:8013` (default host and port).
 
+In-process resolver can also work in an offline mode.
+To enable this mode, you should provide a valid flag configuration file with the option `offlineFlagSourcePath`.
+
+```
+  OpenFeature.setProvider(new FlagdProvider({
+      resolverType: 'in-process',
+      offlineFlagSourcePath: './flags.json',
+  }))
+```
+
+Offline mode uses `fs.watch` to watch for changes to the file, which has (a few caveat)[https://nodejs.org/docs/latest/api/fs.html#caveats] to be aware of.
```

Comment:
```suggestion
Offline mode uses `fs.watch` to watch for changes to the file, which has [a few caveat](https://nodejs.org/docs/latest/api/fs.html#caveats) to be aware of.
```
It might be worth highlighting that this won't work on Docker or K8s where network file systems are used.

Reply:
I wonder if I should just switch to `fs.watchFile`. It uses polling to overcome some of these limitations. The overhead should be minimal, and users should only enable this feature if they understand the implication.

What do you think @Kavindu-Dodanand @toddbaert?

Reply:
I prefer polling with `fs.watchFile` as this should reliably emit flag changes (per documentation and other sources). I would not worry about performance as we recommend the in-process offline mode as an alternative to other well-established flag sync mechanisms we support.